src/main/java/controller/MainController.java

```diff
@@ -156,9 +164,10 @@ private void initMainFrame() {
 
 	/**
 	 * Formats all data to a valid format to a JTable and returns a JTable with the
-	 * cell's content
```

Comment:
Isn't this "cell's content" the correct one? Instead of "cells' content"

Reply:
no, because it isn't a singular cell but a group of cells